GDSCommon-Demo/GDSCommon-Demo/DemoViewController.swift

```diff
@@ -1,21 +1,54 @@
 import GDSCommon
 import UIKit
 
-class DemoViewController: UIViewController {
+class DemoViewController: UIViewController, UIPopoverPresentationControllerDelegate {
     
     let tableview: UITableView = {
         let tv = UITableView()
         tv.backgroundColor = UIColor.systemBackground
         tv.translatesAutoresizingMaskIntoConstraints = false
         return tv
     }()
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
         self.navigationController?.navigationItem.title = "Demo App"
+        self.navigationItem.rightBarButtonItem = .init(image: UIImage(systemName: "ellipsis.circle"),
+                                                       style: .plain,
+                                                       target: self,
+                                                       action: #selector(openDetailView(_:)))
     }
-
+    
+    @objc func openDetailView(_ sender: Any) {
+        let mockItem = MockPopoverItemViewModel(title: "First Item",
+                                                titleFont: .body,
+                                                icon: "1.circle",
+                                                tint: .accent) {
+            print("first item tapped")
+        }
+        
+        let mockItem2 = MockPopoverItemViewModel(title: "Second Item",
+                                                titleFont: .body,
+                                                icon: "2.circle",
+                                                tint: .accent) {
+            print("second item tapped")
+        }
+        let vc = PopoverTableViewController(items: [mockItem, mockItem2])
```

Comment:
Would it be better to name this itemViewModels? think it makes it clear what is being passed in